Devscord.DiscordFramework/Framework/Commands/Responses/responses-configuration.json

```
@@ -34,5 +34,9 @@
   {
     "OnEvent": "CurrentVersion",
     "Message": "```Obecna wersja: {{version}}```"
+  },
+  {
+    "OnEvent": "PrintHelp",
+    "Message": "Dostępny komendy: {{help}}" 
```

Masz literówke
```suggestion
    "Message": "Dostępne komendy: {{help}}" 
```

a poza tym hmm
tu bym sie zastanowił nad tym, czy dałoby sie z poziomu tej konfiguracji ustawić w jaki sposób konfigurować wygląd samego helpa
np opcja napisania wzoru dla każdej linijki 

przykład
```
@(foreach helpInfos as helpInfo)
    {{helpInfo.DefaultPrefix}}{{helpInfo.CommandName}} => {{helpInfo.Description}}\r\n
    \t{{helpInfo.Example}}
```

tak teraz myśle czy warto tworzyć tu własny język szablonów
czy użyć czegoś gotowego, jak np razor albo twig